proto/defs.bzl

```diff
@@ -64,9 +40,11 @@ def ts_proto_library(name, proto, **kwargs):
     )
 
     # Transform .proto files to a single _pb.js file named after the macro
-    pbjs(
+    bin.pbjs(
         name = js_target,
-        data = [":" + proto_target] + _PROTOBUFJS_CLI_DEPS,
+        srcs = [":" + proto_target],
+        copy_srcs_to_bin = False,
+        chdir = "../../../",
```

Comment:
could pass

```
env = {
   "BAZEL_BIN": "."
},
```

instead of the `chdir` I believe; tho I think we should add an explicit attribute for this situation such as `run_out_of_execroot = True` or make it an opt-out `run_out_of_bazel_bin = False`

Reply:
Ah right. Thanks!